pkg/api/handlers/compat/images_build.go

```diff
@@ -580,19 +579,8 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		pullPolicy = buildahDefine.PolicyMap[query.PullPolicy]
 	} else {
 		if _, found := r.URL.Query()["pull"]; found {
```

Comment:
@Luap99 Is this `found` guard needed? Why?

Reply:
I mean if the pull param is not present then quar.yPull is false anyway right?

Reply:
Yes I don't think that is needed and should work without it.